src/neural/loader.cc

```diff
@@ -24,20 +24,35 @@
 #include <fstream>
 #include <iostream>
 #include <sstream>
+#include <string>
 #include "utils/commandline.h"
 #include "utils/exception.h"
 #include "utils/filesystem.h"
+#include "proto/net.pb.h"
+#include "version.inc"
 
-namespace lczero {
+namespace {
```

Comment:
anonymous namespace should be inside named:
```cpp
namespace {
namespace lczero {

// functions

} // namespace

// public functions

} // namespace lczero
```

That way they don't have to explicitly do `lczero::` everywhere.